AssertionError with adding a new Tokenizer!

- I'm trying to add a new `Tokenizer` and was trying to understand how can I do that. I saw the example mentioned in documents and while trying that approach, I got following error. Any suggestion on adding a new Tokenizer?
### Code:

```python
from typing import List

class MyTokenizer:
        
    @staticmethod
    def to_sentences(text: str) -> List[str]:
        return [s.strip() for s in text.split(".")]

    @staticmethod
    def to_words(sentence: str) -> List[str]:
        return [w.strip() for w in sentence.split(" ")]

text = """New York (CNN)When Liana Barrientos was 23 years old, she got married in Westchester County, New York.
A year later, she got married again in Westchester County, but to a different man and without divorcing her first husband."""

parser = PlaintextParser.from_string(text, MyTokenizer)
parser.document
```

### Error:
```python
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-58-02cd6843c524> in <module>
----> 1 parser.document

~/anaconda3/lib/python3.6/site-packages/sumy/utils.py in decorator(self)
     51 
     52         if not hasattr(self, key):
---> 53             setattr(self, key, getter(self))
     54 
     55         return getattr(self, key)

~/anaconda3/lib/python3.6/site-packages/sumy/parsers/plaintext.py in document(self)
     73                 current_paragraph.append(line)
     74 
---> 75         sentences = self._to_sentences(current_paragraph)
     76         paragraphs.append(Paragraph(sentences))
     77 

~/anaconda3/lib/python3.6/site-packages/sumy/parsers/plaintext.py in _to_sentences(self, lines)
     96         if text:
     97             sentences = self.tokenize_sentences(text)
---> 98             sentence_objects += map(self._to_sentence, sentences)
     99 
    100         return sentence_objects

~/anaconda3/lib/python3.6/site-packages/sumy/parsers/plaintext.py in _to_sentence(self, text)
    101 
    102     def _to_sentence(self, text):
--> 103         assert text.strip()
    104         return Sentence(text, self._tokenizer)

AssertionError: 
```

--- a/sumy/parsers/parser.py
+++ b/sumy/parsers/parser.py
@@ -35,7 +35,7 @@ class DocumentParser(object):
         self._tokenizer = tokenizer
 
     def tokenize_sentences(self, paragraph):
-        return self._tokenizer.to_sentences(paragraph)
+        return [s for s in self._tokenizer.to_sentences(paragraph) if s.strip()]
 
     def tokenize_words(self, sentence):
         return self._tokenizer.to_words(sentence)

--- a/sumy/parsers/plaintext.py
+++ b/sumy/parsers/plaintext.py
@@ -84,8 +84,7 @@ class PlaintextParser(DocumentParser):
         for line in lines:
             if isinstance(line, Sentence):
                 if text:
-                    sentences = self.tokenize_sentences(text)
-                    sentence_objects += map(self._to_sentence, sentences)
+                    sentence_objects.extend(self._to_sentence_objects(text))
 
                 sentence_objects.append(line)
                 text = ""
@@ -94,11 +93,9 @@ class PlaintextParser(DocumentParser):
 
         text = text.strip()
         if text:
-            sentences = self.tokenize_sentences(text)
-            sentence_objects += map(self._to_sentence, sentences)
+            sentence_objects.extend(self._to_sentence_objects(text))
 
         return sentence_objects
 
-    def _to_sentence(self, text):
-        assert text.strip()
-        return Sentence(text, self._tokenizer)
+    def _to_sentence_objects(self, text):
+        return (Sentence(s, self._tokenizer) for s in self.tokenize_sentences(text))

